App/src/app/BookingArea.java

```diff
@@ -45,8 +58,8 @@ public BookingArea(String username) {
             ex.printStackTrace();
         }
         String[] city_names = cities.toArray(new String[0]);  
-        for(int i=0;i<city_names.length;i++)
-            System.out.println(city_names[i]);
+        //for(int i=0;i<city_names.length;i++)
+        //    System.out.println(city_names[i]);
```

Comment:
Why do you need to comment this file?

Reply:
Commented because it only outputs the names to the console that were added. you can add it back but it just made the report messy

Reply:
Okay then 👍🏻